plantuml/LinkLoss/init.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+skinparam defaultTextAlignment center
+
+|App|
+start
+
+|Link Loss Service|
+:init;
+
+:construct gatt service for lls;
+
+:set write auth. callback for alert_level_char.;
+
+|Gatt Server|
+:add lls to gatt server;
+-> error;
```

Comment:
so what happens on error?
maybe init should return ble_error_t

Reply:
This crossed my mind too whilst writing the PR.